src/frontend/components/header/header.ts

```diff
@@ -40,12 +40,22 @@ export class Header {
     }
   }
 
+  resetTheme() {
+    this.settingOpened = false;
+    document.removeEventListener('click', this.resetTheme.bind(this), true);
+  }
+
+  openSettings() {
```

Comment:
correct we can but I won't do it on one component we can create an issue to do it for all the components just to be consistent in the code